client/test/backend.go

```diff
@@ -250,7 +288,7 @@ func (b *MockBackend) checkState(s *channel.State, op checkStateFunc) error {
 }
 
 // Withdraw withdraws the channel funds.
-func (b *MockBackend) Withdraw(_ context.Context, req channel.AdjudicatorReq, subStates channel.StateMap) error {
+func (b *MockBackend) Withdraw(_ context.Context, req channel.AdjudicatorReq, subStates channel.StateMap, acc wallet.Address) error {
```

Comment:
Just for my comprehension: This change is necessary to have similar behavior as with the Ethereum backend when using the adjudicator's receiver address to change the address the funds are sent to, correct?

Reply:
Exactly. In fact, it was also necessary to adapt the Balance Reader interface, because of that.